### ISSUE TITLE
Accessibility: Color Contrast Header Links

### ISSUE THREAD

**Issue by danielgroner** [xz/new.css#24]
The links <a> in the <header> fail some contrast ratio accessibility tests, for example Lighthouse Audit in Chrome Dev Tools. Might be a good thing to tweak this.

**Comment by halfmage** [comment]
Yeah I think this is a good idea! Could you maybe document exactly where it fails the accessibility tests?

**Comment by danielgroner** [comment]
As far as I see it's just the lightblue links in the header, if the header has a lightgrey background. I hope that helps. 
You can check it if you open chrome dev tools, go to audits tab and select the "accessibility" checkbox, then generate the report.
